spec/support/shared_examples/validation_examples.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
My other comment notwithstanding, these are really nicely done, you're getting pretty awesome at spec support stuff :smile:.